websites/I/Indavideó/presence.ts

```diff
@@ -0,0 +1,104 @@
+const presence = new Presence({
+		clientId: "1234605936098807829",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+let searchBar: string,
+	videoTitle: string,
+	uploader: string,
+	paused: boolean,
+	currentTime: number,
+	duration: number;
+
+presence.on(
+	"iFrameData",
+	(data: { currentTime: number; paused: boolean; duration: number }) => {
+		({ currentTime, paused, duration } = data);
+	}
+);
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "https://i.imgur.com/WXPi1xl.png",
+			startTimestamp: browsingTimestamp,
+		},
+		strings = await presence.getStrings({
+			play: "general.playing",
+			pause: "general.paused",
+		}),
+		{ hostname, href, pathname } = document.location;
+	if (pathname === "/" && hostname === "indavideo.hu")
+		presenceData.details = "On the homepage";
+	else if (
+		pathname.includes("/search/text/") ||
+		pathname.includes("/search/detailed/")
+	) {
+		searchBar = (<HTMLInputElement>document.querySelector("#ddl_search-what"))
+			.value;
+		presenceData.details = `Searching for "${searchBar}"...`;
+	} else if (pathname.includes("/video/")) {
+		delete presenceData.startTimestamp;
+		(videoTitle = document.querySelector(".vid_header_title").textContent),
+			(uploader = document.querySelector(".vid_header_username").textContent);
```

Comment:
Don't use `,` when you (re)assign let variables.
```suggestion
		videoTitle = document.querySelector(".vid_header_title").textContent;
			uploader = document.querySelector(".vid_header_username").textContent;
```

Reply:
Fixed.